src/Commands/BuildToolsBase.php

```diff
@@ -78,8 +78,12 @@ public function providerManager()
 
     protected function createProviders($git_provider_class_or_alias, $ci_provider_class_or_alias)
     {
-        $this->ci_provider = $this->providerManager()->createProvider($ci_provider_class_or_alias, \Pantheon\TerminusBuildTools\ServiceProviders\CIProviders\CIProvider::class);
-        $this->git_provider = $this->providerManager()->createProvider($git_provider_class_or_alias, \Pantheon\TerminusBuildTools\ServiceProviders\RepositoryProviders\GitProvider::class);
+        if (isset($ci_provider_class_or_alias)) {
```

Comment:
Let's refactor `createProviders` into `createGitProvider` and `createCIProvider` so that we do not need to pass 'null' in for the parameters. `createProvilders` can remain, and call through to the new functions.

Reply:
Done.